app/templates/adf-cli-apa-acs-template/src/app/stencils.module.ts

```diff
@@ -1,44 +1,26 @@
 /* tslint:disable */
 
-import { Component, NgModule } from '@angular/core';
-import { WidgetComponent } from '@alfresco/adf-core';
+import { Component, NgModule } from "@angular/core";
+import { WidgetComponent } from "@alfresco/adf-core";
 
 @Component({
-    selector: 'custom-editor',
-    template: `
-        <div style="color: red">Look, I'm a custom editor!</div>
-    `
+  selector: "custom-editor",
+  template: ` <div style="color: red">Look, I'm a custom editor!</div> `,
 })
 export class CustomEditorComponent extends WidgetComponent {
-
-    constructor() {
-        super();
-    }
 }
 
 @Component({
-    selector: 'custom-stencil-01',
-    template: `<div style="color: green">ADF version of custom Activiti stencil</div>`
+  selector: "custom-stencil-01",
+  template: `<div style="color: green">
+    ADF version of custom Activiti stencil
+  </div>`,
```

Comment:
trailing commas intentionally?

Reply:
those were part of the old linter, should be ok but I've cleaned anyway, thanks